packages/mobile/src/components/audio/RNVideoAudioPlayer.tsx

```diff
@@ -260,7 +262,7 @@ export const RNVideoAudioPlayer = () => {
           queryParams = await getQueryParams({
             audiusBackendInstance,
             nftAccessSignature,
-            userId: currentUserId
+            userId: currentUserId || undefined
```

Comment:
Is this due to `useSelector` potentially returning `null` and the type `getQueryParams` only supporting `undefined`?
Generally would prefer not to do boolean assignment with undefined like this :-). Usually feels like it means types are incompatible somewhere.

Reply:
Yeah actually I think this was unnecessary 👍